NUAttribute.js

```diff
@@ -39,6 +42,7 @@ export default class NUAttribute extends NUObject {
             maxLength: obj.maxLength || -1,
             minLength: obj.minLength || -1,
             remoteName: remName,
+            subType: obj.subType,
```

Comment:
do you use this in anyway? list validation should be done based on this subtype. We need to support as well json objects as types. We have this for uplink/br connections.

Reply:
Right now, no. This PR just includes it in the model, intention is to just fit in the monolithe-generated model. Validation is to be added, may be in a separate PR.